src/hooks/useENSAvatar.ts

```diff
@@ -134,11 +135,13 @@ function useERC1155Uri(
   const contract = useERC1155Contract(contractAddress)
   const balance = useSingleCallResult(contract, 'balanceOf', accountArgument)
   const uri = useSingleCallResult(contract, 'uri', idArgument)
+  // ERC-1155 allows a generic {id} in the URL, so prepare to replace if relevant
+  const idHex = id ? BigNumber.from(id).toHexString().substring(2) : id
```

Comment:
i think it has to be padded to 64 characters

Reply:
Added leading zero pad to 64 hex chars